src/js/components/Image/doc.js

```diff
@@ -26,6 +26,9 @@ export const doc = Image => {
       PropTypes.string,
       PropTypes.bool,
     ]).description('Transparency of the image.'),
+    fill: PropTypes.bool.description(
```

Comment:
I'm inclining for consistency of the fill prop.
When `fill` is defined on other components it allows the values of 'vertical' | 'horizontal' | boolean.
It's been done a few days ago in a similar way for WorldMap.
Do you feel like it is needed for Image as well?

Reply:
Sure thing! I had referred to [https://github.com/grommet/grommet/issues/3222#issuecomment-516176939](url) to assume that `fill` should assumes only `true` or `false`. Should all the values be supported?

Reply:
I think we're OK to just have it be a boolean for now.